spec.bs

```diff
@@ -916,9 +929,13 @@ as appropriate:
     [=upon rejection=] of |promise|, run the following steps:
     1. Let |globalScopes| be |this|'s [=Worklet/global scopes=].
     1. [=Assert=]: |globalScopes|' [=list/size=] equals 1.
-    1. Set |globalScopes|[0]'s {{SharedStorageWorkletGlobalScope/
-        privateAggregation}}'s [=PrivateAggregation/scoping details=] a new
-            [=scoping details=] with the items:
+    1. Let |privateAggregationObj| be |globalScopes|[0]'s
+        {{SharedStorageWorkletGlobalScope/privateAggregation}}.
+    1. If [=this=]'s [=relevant global object=]'s [=associated document=] is
```

Comment:
Optional: We can probably just set this field to the permission policy check without checking the value.

Reply:
Done